Automattic/jetpack
# Analyzer: Handle more code tokens

This PR adds support for some missing cases, which was preventing the script from running against some of the plugins

#### Jetpack product discussion


pd5faL-1O-p2
#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* It is a bit hard to test it
* Download Facebook to Woo plugin: https://wordpress.org/plugins/facebook-for-woocommerce/
* Extract it somewhere.
* run below script from within `packages/analyzer/scripts` folder:
```
require dirname( __DIR__ ) . '/vendor/autoload.php';
$path_to_plugin = "/abs/path/to/plugin";
$invocations = new Automattic\Jetpack\Analyzer\Invocations();
$invocations->scan( $path_to_plugin);
```
* Make sure there is no warning or notices generated
